common/coat_of_arms/coat_of_arms/wc_dynasties.txt

```diff
@@ -133,4 +143,62 @@ house_burningblade = d_burning_blade
 #		texture = "ce_mechagon.dds"
 #		color1 = "purple"
 #	}
-#}
\ No newline at end of file
+#}
+
+### Dwarfs ###
+
+4010 = { # Thaurissan
+	pattern = "pattern_solid.dds"
+	color1 = rgb { 123 105 91 }
+	colored_emblem = {
+		texture = "ce_anvilrage.dds"
+		color1 = rgb { 66 48 31 }
+		color3 = rgb { 139 76 46 }
+	}
+}
+
+4200 = { # Bronzebeard
+	pattern = "pattern_solid.dds"
+	color1 = rgb { 68 58 49 }
+	colored_emblem = {
+		texture = "ce_dwarf_golem_crown.dds"
+		color1 = rgb { 205 127 50 }
+		color2 = rgb { 225 102 22 }
+		color3 = rgb { 67 70 75 }
+	}
+}
+
+4235 = { # Stormpike
+	pattern = "pattern_solid.dds"
+	color1 = rgb { 64 74 127 }
+	colored_emblem = {
+		texture = "ce_hammer_and_anvil.dds"
+		color1 = rgb { 147 101 37 }
+	}
+}
+
+### Trolls ###
+
+8220 = { # Zilzikyo 
+	pattern = "pattern_solid.dds"
+	color1 = rgb { 40 78 36 }
+	colored_emblem = {
+		texture = "ce_troll_mask_zuljin.dds"
```

Comment:
I would make it wider, it's barely recognazible.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/121759992-7adfe500-cb39-11eb-8963-78f6d137cc17.png)

</details>